### DISCUSSION TITLE
[Mobile] Flutter-Quill editor inside a scrollable form

### DISCUSSION THREAD

**Discussion by niccord** [singerdmx/flutter-quill#291] [Mobile] [answered]
My issue is about [Mobile]

I would like to put my `QuillEditor` inside my form and not on a dedicated page of my app.

I've tried with a `Column` inside a `SingleChildScrollView` and with a `ListView` widget, but the result is the same:
- since there is no text, the editor is small and I would like it to be "expanded" (I know it can't be as such inside a scroll, but you get the point);
- the overflow of the text does not make the page scroll accordingly, so the users can't see what they are typing.

Here's some code:
``` dart
return Scaffold(
  appBar: AppBar(
    title: Text("Reply form"),
    actions: [
      IconButton(
        onPressed: () {},
        icon: Icon(Icons.send),
        tooltip: "Send",
      )
    ],
  ),
  body: SafeArea(
    child: Form(
      child: ListView(
        children: [
          Text("From: test@my.org"),
          SizedBox(height: 8),
          chipInputField(
            label: "To",
            onChanged: (List<Object?> o) {},
            initialValue: ["first@email.it"],
          ),
          chipInputField(
            label: "Cc",
            onChanged: (List<Object?> o) {},
            initialValue: ["second@email.it"],
          ),
          chipInputField(
            label: "Bcc",
            onChanged: (List<Object?> o) {},
            initialValue: ["third@email.it"],
          ),
          QuillEditor(
            controller: quillController,
            scrollable: true,
            scrollController: ScrollController(),
            focusNode: FocusNode(),
            padding: EdgeInsets.all(5),
            autoFocus: true,
            readOnly: false,
            expands: false,
            placeholder: "compose_email",
          ),
          QuillToolbar.basic(
            controller: quillController,
            showUnderLineButton: false,
            showStrikeThrough: false,
            showColorButton: false,
            showBackgroundColorButton: false,
            showListCheck: false,
            showIndent: false,
          ),
        ],
      ),
    ),
  ),
);
```

First issue: the compose email part is not expanded inside the application page

[![the compose email part is not expanded inside the application page][1]][1]

Second issue: as long as I write some text, the widget is not scrolling

[![the keyboard is over the text and the widget is not scrolling][2]][2]

  [1]: https://i.stack.imgur.com/kZMQg.png
  [2]: https://i.stack.imgur.com/j3SAa.png

**Comment by kevinDespoulains** [comment, marked as answer]
Hello,

I have created a PR https://github.com/singerdmx/flutter-quill/pull/371 to solve this issue with a code example, can you take a look and tell me if it is working for you ? 

Thank you !
